spec/spec.tex

```diff
@@ -1,4 +1,4 @@
-\documentclass[10pt,twoside,titlepage]{spec}
+\documentclass[10pt,oneside,titlepage]{spec}
```

Comment:
Lines 116-118.  I don't think you need both evensidemargin and oddsidemargin.  I think oddsidemargin will do.  You can set the textwidth to 6.5in so the margin is the same size on each side of the page (1.0in).  The default twosided layout leaves an extra 1/2 inch at the spine side of the page, for binding....